scarb/src/compiler/helpers.rs

```diff
@@ -51,13 +51,23 @@ pub fn build_compiler_config<'c>(
             diagnostics_reporter
         },
         replace_ids: unit.compiler_config.sierra_replace_ids,
+        inlining_strategy: unit.compiler_config.inlining_strategy.clone().into(),
         add_statements_functions: unit
             .compiler_config
             .unstable_add_statements_functions_debug_info,
         ..CompilerConfig::default()
     }
 }
 
+impl From<InliningStrategy> for cairo_lang_lowering::utils::InliningStrategy {
```

Comment:
random suggestion: I think it'd be awesome to also have an

```rust
#[allow(unused)]
impl From<cairo_lang_lowering::utils::InliningStrategy> for InliningStrategy
```

so that the compiler will yell if something is added upstream and forgotten here

Reply:
Yes, thanks! Added